core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -482,7 +485,7 @@ class TestCaseWriter {
 
         if (res.getBodyType() != null) {
             val type = res.getBodyType()!!
-            if (type.isCompatible(MediaType.APPLICATION_JSON_TYPE)) {
+            if (type.isCompatible(MediaType.APPLICATION_JSON_TYPE) || type.toString().contains("+json")) {
```

Comment:
i think it should ignore the case, eg something like `type.toString().toLowerCase().contains("+json")` as I think it should handle `+JSON` as well

Reply:
Done.